### ISSUE TITLE
decoder result is strange

### ISSUE THREAD

**Issue by luweishuang** [WING-NUS/sequicity#17]
i trained on the "camrest" dataset and do the test, generated the result files as 
[camrest-rl.csv.zip](https://github.com/WING-NUS/sequicity/files/2754478/camrest-rl.csv.zip)
in this file,the generated respon wrote as "there are no chinese restaurants in the area_SLOT part of town . would you like their phone number is phone_SLOT . would you like their phone number is phone_SLOT . would you like their phone number is phone_SLOT . would" . it is too long and it trend to repeat and i found its part structure like "there are no *** matching your request" appeared in many turn's response. i ran "python3 model.py -mode train -model tsdf-camrest" and "python3 model.py -mode test -model tsdf-camrest", is there anything wrong with this steps or this results in normal for sequicity

**Comment by AuCson** [comment]
Hi,
No, it is not normal. Please use '-adjust' option to tune the parameters with a small learning rate (0.0003). Thanks!

**Comment by luweishuang** [comment]
i tried the "adjust" mode with lr=0.0003 and it did solve the repeated problem. but i still found the generated response stated a different mean comparing to response, like 
[camrest-rl.csv.zip](https://github.com/WING-NUS/sequicity/files/2757852/camrest-rl.csv.zip)
can you give some opinions?

**Comment by luweishuang** [comment]
i set degree_input_new = torch.FloatTensor(degree_input.size()).zero_() and do the eval()  function, found the generated_response have no influences at all.  
[camrest-rl.csv.zip](https://github.com/WING-NUS/sequicity/files/2758150/camrest-rl.csv.zip)

**Comment by AuCson** [comment]
Hi,
It is natural that generated responses are less diverse than ground truth. 
I find that you are not using beam search. Using beam search will increase response diversity.
